EX2/6.4_remember_remember.py

```diff
@@ -0,0 +1,30 @@
+from PIL import Image
+
+
+def find_secret(image_path):
```

Comment:
Use type hints and return types -
https://www.geeksforgeeks.org/duck-typing-in-python/